python_ta/allow_pylint_comments_false_testcase.py

```diff
@@ -0,0 +1,14 @@
+def some_function():
```

Comment:
Okay great, so you should turn these into proper test cases under the `tests` folder. Take a look at how existing configuration tests have been set up (you might also ask your PythonTA team members who have done some similar testing)

Reply:
You should now delete this file, and the other old testcase file you added (below).